NodeJS/Week-1/avg.js

```diff
@@ -0,0 +1,11 @@
+// avg.js
+if (process.argv.length <= 2) {
+    console.log("Please provide numbers as arguments.");
+    process.exit(1);
+}
+
+let numbers = process.argv.slice(2).map(Number);
+let sum = numbers.reduce((acc, val) => acc + val, 0);
+let avg = sum / numbers.length;
+
+console.log(`The average is: ${avg}`);
```

Comment:
nicely done! you use `let` for all your variables? Why not `const`? Always use `const` until the program won't `let` you.

you can put the logic of this file into a function.